docs/docs/rest-api/public/api/v2/types/pathId.raml

```diff
@@ -0,0 +1,4 @@
+#%RAML 1.0 DataType
+type: string
+pattern: ^(\/?((\.{2})|([a-z0-9][a-z0-9\-.]*[a-z0-9]+)|([a-z0-9]*))($|\/))+$
```

Comment:
not sure why we allow segments that contain `..` @meichstedt is there some history here?